test/features/calendar.feature

```diff
@@ -36,10 +36,20 @@ Feature: calendar module
         And make a POST request to "/calendar" with:
             | test | "test" |
         Then the response status code should be 400
-        And the response should contains:
-            | statusCode | 400                                                                                                                                                                                                                                                                                                                                              |
-            | message    | ["name must be a string","name should not be empty","platform must be one of the following values: YouTube, Twitch","platform must be a string","platform should not be empty","author should not be empty","startDate should not be empty","startDate must be a Date instance","endDate must be a Date instance","endDate should not be empty"] |
-            | error      | "Bad Request"                                                                                                                                                                                                                                                                                                                                    |
+        And the response should contain:
+            | statusCode | 400           |
+            | error      | "Bad Request" |
+        And the response property "message" has items:
+            | name must be a string                                         |
```

Comment:
We should document somewhere that we dont use quotes here.

Reply:
Good point 👍 

We should probably document the whole testing process with some simple examples - I will raise an issue

Reply:
Issue created https://github.com/EddieHubCommunity/api/issues/82